### ISSUE TITLE
Remove "google|gmail" from trustedViewerHosts in config.js

### ISSUE THREAD

**Issue by choumx** [ampproject/amphtml#25040]
Context: https://github.com/ampproject/amphtml/pull/25027#issuecomment-541469996

**Comment by choumx** [comment]
@mrjoro FYI this helps decouple "google" from our source code.

@ampproject/wg-infra Is this related to any planned OpenJS migration work?

**Comment by rsimha** [comment]
Yes, this will be included in the list of tasks for the Open JS migration. There's a tracking issue at https://github.com/ampproject/wg-infra/issues/18 with a larger discussion that includes folks from @ampproject/wg-caching.

**Comment by rsimha** [comment]
_[Infra triage]_ Since this issue was filed two years ago, the OpenJS migration was completed. See https://github.com/ampproject/wg-caching/issues/13#issuecomment-597297041 for more details.

However, the offending lines are still in our codebase.

https://github.com/ampproject/amphtml/blob/e5c57a647118dfff7f6a2e81bcf5a624550504f1/src/config.js#L97-L112

Assigning to @jridgewell for comment on if / how they can be removed from AMP.
